src/components/ListOfTodo/ListOfTodo.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useEffect, useState } from 'react';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import { Todo } from '../../types/Todo';
+import { TodoStatus } from '../../types/TodoStatus';
+import { InfoTodo } from '../InfoTodo';
+import { FilterForTodo } from '../FilterForTodo';
+import { deleteTodo } from '../../api/todos';
+import '../../App.css';
+// eslint-disable-next-line import/order
+import classNames from 'classnames';
+
+type Props = {
+  todos: Todo[];
+  handleLoadTodos: () => void;
+  query: string;
+  isEditing: boolean;
+};
+
+export const ListOfTodo: React.FC<Props> = ({
+  todos,
+  handleLoadTodos,
+  query,
+  isEditing,
+}) => {
+  const [filteredTodos, setFilteredTodos] = useState<Todo[]>([]);
+  const [currentStatus, setTodosStatus] = useState<TodoStatus>(TodoStatus.ALL);
+
+  const filterTodos = () => {
+    let filtered;
+
+    switch (currentStatus) {
+      case TodoStatus.ACTIVE:
+      case TodoStatus.COMPLETED:
+        // eslint-disable-next-line max-len
+        filtered = todos.filter((todo) => (currentStatus === TodoStatus.ACTIVE ? !todo.completed : todo.completed));
+        break;
+      default:
+        filtered = todos;
+    }
+
+    return filtered;
+  };
```

Comment:
Actually, I think here you no need to create variable out of switch block, instead you  can use return statement inside switch